### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by vinsterr** [thmspry/memrise_bot#1]
![image](https://github.com/thmspry/memrise_bot/assets/103316082/f9eeee6e-705b-4934-b629-11285c7b6c9d)
Any ideas on how to fix this?
Thanks

**Comment by thmspry** [comment]
I think you forgot to write your Memrise username and password in the `.env` file, as mentionned in the [Deployment section of the README](https://github.com/thmspry/memrise_bot#deployment).

**Comment by vinsterr** [comment]
> I think you forgot to write your Memrise username and password in the `.env` file, as mentionned in the [Deployment section of the README](https://github.com/thmspry/memrise_bot#deployment).


**EDIT**

I managed to overcome that error, however once logged in the window crashes and now brings up this error. I  have altered the delay values greatly, but can't see that being the issue as my internet is fairly good.
![image](https://github.com/thmspry/memrise_bot/assets/103316082/985fc7d4-f15d-4b9b-9fc6-d565f26a06d0)

Thanks again

**Comment by thmspry** [comment]
Yes sorry, I updated the repository yesterday, due to a change in the URL format of the website that was causing this error. You can now update your version of the repository, it should work.

**Comment by vinsterr** [comment]
> Yes sorry, I updated the repository yesterday, due to a change in the URL format of the website that was causing this error. You can now update your version of the repository, it should work.

![image](https://github.com/thmspry/memrise_bot/assets/103316082/653a78ba-6b19-4e56-837c-1d2714d1f329)
After downloading the updated repo, it now returns this error, once on the course and showing the words.
Thanks

**Comment by thmspry** [comment]
Back to back ! Please check for the new TypeScript migration, a lot of thing changed
